src/interface/src/app/shared/nav-bar/nav-bar.component.ts

```diff
@@ -8,4 +9,14 @@ import { Component, EventEmitter, Input, Output } from '@angular/core';
 export class NavBarComponent {
   @Input() breadcrumbs: string[] = [];
   @Output() goBack = new EventEmitter<void>();
+
+  constructor(@Inject(WINDOW) private window: Window) {}
+
+  copyLink() {
+    console.log('not implemented');
+  }
+
+  print() {
+    this.window.print();
```

Comment:
Noticed that as of now, the map control panel takes up half of the print pdf and you can barely see the map. Just checked in with Michele and it would be preferred if we could just print the map. Is that possible? 

Let me know also if you'd rather tackle that in a future pr

<img width="1792" alt="Screenshot 2023-09-07 at 11 10 35 AM" src="https://github.com/OurPlanscape/Planscape/assets/18537927/b5cf92fe-e0f2-4a28-ab82-8613e2392784">